tutorials/plot_parse.py

```diff
@@ -11,7 +11,7 @@
 Closing this string quotes on same line"""
 
 
-##############################################################################
+# %%
```

Comment:
I would add a note that at the top of the file that 2 types of separators are possible.  Say you'll use
the ### at the beginning of the file, put a note saying that you'll use in what follows the alternative
approach with # %%

does it make sense?

Reply:
Yes I realised above when you talked about the double up  that the plot_parse.py file is actually a page in the sphinx-gallery documentation (whoops). Am amending this page to add the relevant information.